### DISCUSSION TITLE
Error structure refactoring

### DISCUSSION THREAD

**Discussion by jordanjones243** [AlaskaAirlines/auro-formvalidation#17] [General]
### General Support Request

There needs to be a conversation around the error priorities in form validation. There are many cases that people have ran into where multiple errors from component validation has come up, yet we have no specific guidelines about what errors should be handled first.

### Possible Solution

Investigate how HTML5 components handle error priority and have a discussion about the structure of our form validation compared to that.

**Comment by blackfalcon** [comment]
@jordanjones243 can you please update this issue with the outcomes of that meeting and either close or append to other work in progress.
